features/in_onderzoek.feature

```diff
@@ -1080,31 +1081,31 @@ Functionaliteit: in onderzoek
-      | huisnummer                | 081120 | VerblijfplaatsOnbekend | true        |             |           |        |            |                   |                    |                        |                   |                   |                       |                  | 
-      | woonplaats                | 081170 | VerblijfplaatsOnbekend |             | true        |           |        | true       |                   |                    |                        |                   | 2022-03-07        | Datum                 | 7 maart 2022     | 
-      | locatiebeschrijving       | 081210 | VerblijfplaatsOnbekend | true        |             |           |        |            | true              | 2022-03-07         | Datum                  | 7 maart 2022      |                   |                       |                  | 
-      | adresregel1               | 081330 | VerblijfplaatsOnbekend | true        |             |           |        |            |                   |                    |                        |                   |                   |                       |                  | 
-      | groep gemeente            | 080900 | VerblijfplaatsOnbekend |             | true        |           |        |            |                   |                    |                        |                   |                   |                       |                  | 
-      | groep adres               | 081100 | VerblijfplaatsOnbekend | true        | true        | true      | true   | true       | true              | 2022-03-07         | Datum                  | 7 maart 2022      | 2022-03-07        | Datum                 | 7 maart 2022     | 
-      | groep locatiebeschrijving | 081200 | VerblijfplaatsOnbekend | true        |             |           |        |            | true              | 2022-03-07         | Datum                  | 7 maart 2022      |                   |                       |                  |
-      | groep verblijf buitenland | 081300 | VerblijfplaatsOnbekend | true        | true        |           |        |            | true              | 2022-03-07         | Datum                  | 7 maart 2022      |                   |                       |                  |
-      | categorie verblijfplaats  | 080000 | VerblijfplaatsOnbekend | true        | true        | true      | true   | true       | true              | 2022-03-07         | Datum                  | 7 maart 2022      | 2022-03-07        | Datum                 | 7 maart 2022     | 
+      | gegeven in onderzoek      | waarde | type                   | adresregel1 | adresregel2 | korteNaam | officieleStraatnaam | woonplaats | verbl.plaats type | verbl.plaats datum | verbl.plaats datumtype | verbl.plaats lang | verbl.adres datum | verbl.adres datumtype | verbl.adres lang |
```

Comment:
VerblijfplaatsOnbekend wordt alleen als type geleverd als land code gelijk is aan 0000. In dit geval zou het type Adres moeten zijn